src/handler/BallotHandler.ts

```diff
@@ -32,7 +32,15 @@ export class BallotHandler {
 
   public getBallots = ballotService.getBallots;
 
-  public getRunningBallot = ballotService.getRunningBallot;
+  public getRunningBallot = async () => {
+    const ballot = await ballotService.getRunningBallot();
+
+    if (!ballot) {
+      throw new createHttpError.NotFound("There is no running ballot");
```

Comment:
You can directly use the `NotFound` class like in line 28

Reply:
Resolved in [#52 Directly using NotFound in BallotHandler](https://github.com/LeoTuet/backend-we-want-to-sleep/pull/54/commits/cbc9572279ea49100e3dfb4376532fea2a5da678)